independent-publisher-connectors/Lexica/apiDefinition.swagger.json

```diff
@@ -0,0 +1,150 @@
+                        "description": "The prompt.",
+                        "title": "Prompt"
+                      },
+                      "width": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "The image width.",
+                        "title": "Width"
+                      },
+                      "height": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "The image height.",
+                        "title": "Height"
+                      },
+                      "seed": {
+                        "type": "string",
+                        "description": "The seed.",
+                        "title": "Seed"
+                      },
+                      "grid": {
+                        "type": "boolean",
+                        "description": "Whether this image is a grid of multiple images.",
+                        "title": "Grid"
+                      },
+                      "model": {
+                        "type": "string",
+                        "description": "The model used to generate this image.",
+                        "title": "Model"
+                      },
+                      "guidance": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "The guidance scale.",
+                        "title": "Guidance"
+                      },
+                      "promptid": {
+                        "type": "string",
+                        "description": "The prompt identifier.",
+                        "title": "Prompt ID"
+                      },
+                      "nsfw": {
+                        "type": "boolean",
+                        "description": "Whether NSFW.",
+                        "title": "NSFW"
+                      }
+                    }
+                  },
+                  "title": "images"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search for a term",
+        "description": "Retrieve a list of images generated by the query term(s). For a reverse image search, you can pass in the URL of an image as the query.",
+        "operationId": "Search",
+        "parameters": [
+          {
+            "name": "q",
+            "in": "query",
+            "x-ms-summary": "Query",
+            "description": "The search query.",
+            "required": true,
+            "type": "string"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://lexica.art/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://lexica.art/"

```

Comment:
@troystaylor is it possible to provide a valid privacy policy link? Maybe this one https://discord.com/privacy?

Reply:
Amjed, I cannot find a privacy policy for this service.

Reply:
Is this related? 
![image](https://user-images.githubusercontent.com/97551789/217678905-9f3bb0ec-f64b-4d9e-88bf-d9756587a260.png)

Reply:
No, the service isn’t owned by Discord. That is just a chat channel